### DISCUSSION TITLE
Add context variable to bind doesn't work on windows

### DISCUSSION THREAD

**Discussion by Jue07** [frangoteam/FUXA#1173] [General]
Hello,

I want to add a context variable for binding in my project on Windows but it doesn't work. Unable to display the output. Any solution to this problem?

Thank you

**Comment by unocelli** [comment]
Hi, If you want someone to try to give a good answer, you need to take the time to provide adequate information, such as a system configuration, sample file, the version of os, node, npm, etc..

**Comment by Jue07** [comment]
@unocelli I have found this feature to work using placeholders in the master branch. Thank you.
